### ISSUE TITLE
"FSF Free/Libre" missing for several licenses

### ISSUE THREAD

**Issue by donaldr3** [spdx/license-list-XML#647]
The following licenses are considered free licenses by the Free Software Foundation, but are not marked as such on the list here:

* GPL-1.0-only
* GPL-1.0-or-later
* OFL-1.0
* LGPL-2.0-only
* LGPL-2.0-or-later

For each, the column "FSF Free/Libre" should indicate 'Y'. Thank you.

**Comment by wking** [comment]
This information is not tracked in this repository, it is pulled from [here][1] by the [license list publisher][2].  The FSF-api issue tracking this particular issue is wking/fsf-api#7, and that's currently blocked on the FSF saying something about those licenses on [their license-list page][3].  I suggest closing this issue as a duplicate of wking/fsf-api#7 (but I'm not a maintainer here, and they can do what they want).  If you want to get the issue resolved, I suggest [contacting the FSF][4] and asking them to list the older licenses on [their license-list page][3] under either their "GPL-Compatible Free Software Licenses" or "GPL-Incompatible Free Software Licenses" sections.  They may also want to add entries for those licenses to [this table][5].

[1]: https://github.com/wking/fsf-api
[2]: https://github.com/spdx/LicenseListPublisher
[3]: https://www.gnu.org/licenses/license-list.html
[4]: https://www.fsf.org/about/contact/
[5]: https://www.gnu.org/licenses/gpl-faq.html#AllCompatibility

**Comment by donaldr3** [comment]
Sorry I wasn't clear before, I'm from the FSF. We were told to file this issue here, but if we can fix it by updating things on our end for the API then I think that makes sense, so I'll go ahead and close this here.

**Comment by wking** [comment]
> ... I'm from the FSF...

Ooh :).  Any chance I can talk you into picking up the API yourselves?  There's some discussion about how that might work in wking/fsf-api#18, and I think API consumers would be happier to have the FSF providing canonical information about their positions than they are with my code and GitHub's hosting as go-betweens.

> ... if we can fix it by updating things on our end for the API then I think that makes sense...

As long as I'm still maintaining the API repo, it would be nice to get notified when the FSF updates their list page.  Even if I don't have to adjust the Python, I'd need to rebuild the compiled site to pull in your changes.  Can you file a new wking/fsf-api issue or comment on an existing one when you make changes?  Or is there an email list or other channel through which I can subscribe to see changes?

**Comment by bradleeedmondson** [comment]
> Do we want to block our next point release on FSF activity?
No, that's not what I intended here. I thought we could update your API with these 5 entries (for 3.2), and push for FSF to maintain the list on their own (regardless of our release activity). Is that alright with you?

**Comment by wking** [comment]
> > Do we want to block our next point release on FSF activity?
>
> No, that's not what I intended here. I thought we could update your API with these 5 entries (for 3.2)...

I'm personally fairly sure that the the GPL-1.0 is `libre` but neither `gpl-3-compatible` nor `gpl-2-compatible`.  But I'm not comfortable putting words in the FSF's mouth, so I'm not going to update the API repo to make those claims without some FSF wording to back them up.  When the FSF decides to publish that wording (if ever) is up to them.  I don't see why this aspect would impact our release planning here.

> ... and push for FSF to maintain the list on their own (regardless of our release activity).

I try to hand off the repo whenever the opportunity presents itself (although see wking/fsf-api#18 for some ideas about workflows where the FSF manages the API).  But the FSF will pick it up (or not) in their own time.  I don't see why this aspect would impact our release planning here either.
